smartseq2_single_sample/run_multi_samples/ss2_multi_sample_wf.wdl

```diff
@@ -0,0 +1,98 @@
+import "ss2_single_sample_wf.wdl" as singlesample
+
+task GatherMetrics {
+
+  Array[File]+ input_metrics_fn
```

Comment:
What is the plus for here? Does it require at least one item in the array? I'm not too familiar with how this operator works - I would sort of expect it to mean that we're requiring one or more arrays, which I don't think is what you're trying to do. Can you rename this variable or comment to make it a bit clearer? I think it's an array of metrics files, but the name input_metrics_fn sounds like it's an array of file *names* rather than the files themselves.

Reply:
yes, specific for array, at least one.  It is fine without '+'.  also changed name to 'input_metrics_list'